examples/typing/mg.jl

```diff
@@ -3,6 +3,149 @@ using Cassette
 using LightGraphs
 using MetaGraphs
 
+Cassette.@context TypeCtx
+            
+"""   TypeCtx
+
+creates a MetaDiGraph tracking the types of args and ret values throughout a script
+
+"""
+TypeCtx
+            
+            
+
+"""   FCollector(depth::Int,frame::function,data::FCollector)
+
+struct to collect all the "frames" called throughout a script
+        
+"""
+mutable struct FCollector{I,F,C}
```

Comment:
Do these need to be in this script? Shouldn't they be imported from SemanticModels?